javascript/index.js

```diff
@@ -56,3 +89,30 @@ class BookStore {
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Please you are not allowed to disable any eslint in your project please do well to enable eslint and fix any error locally on your machine before making a PR

Reply:
This change has been done well